extensions/vscode/package.json

```diff
@@ -0,0 +1,134 @@
+{
+  "name": "tutorialkit-vscode",
```

Comment:
I had to rename it from `tutorialkit` to `tutorialkit-vscode` to avoid the collision with the CLI. The build script for publishing the package to the VSCode marketplace rename it to `tutorialkit`.